pyproject.toml

```diff
@@ -86,6 +86,7 @@ lxml = {version = "^4.9.2", optional = true}
 pymupdf = {version = "^1.22.3", optional = true}
 pypdfium2 = {version = "^4.10.0", optional = true}
 gql = {version = "^3.4.1", optional = true}
+pdfplumber = {version = "^0.9.0", optional = true}
```

Comment:
There are other missing pkgs that are used in the code base but aren't appearing in the optional group